website/src/index-page.tsx

```diff
@@ -1,119 +1,374 @@
+                  style={{ stroke: pickGradient(2)[0] }}
+                />
+                <span className="title-font font-medium text-black dark:text-white">
+                  {env}
+                </span>
+              </div>
+            </div>
+          ))}
+        </div>
+      </Feature>
+    </div>
+  )
+}
+
+function FeatureWrapper({ children }: PropsWithChildren) {
+  return (
+    <div
+      className={`
+        w-full py-24
+        odd:bg-gray-50
+        odd:dark:bg-gray-900
+        even:bg-white
+        even:dark:bg-black
+      `}
+    >
+      {children}
+    </div>
+  )
+}
+
+function Feature(
+  props: PropsWithChildren<{
+    title: string
+    description: React.ReactNode
+    highlights?: Array<{
+      title: string
+      description: React.ReactNode
+      icon?: React.ReactNode
+    }>
+    image?: string | StaticImageData
+    gradient: number
+    flipped?: boolean
+  }>,
+) {
+  const { title, description, children, image, gradient, flipped } = props
+  const [start, end] = pickGradient(gradient)
+
   return (
+    <FeatureWrapper>
+      <div className="container box-border px-6 mx-auto flex flex-col gap-y-24">
+        <div
+          className={`flex flex-col gap-24 md:gap-12 lg:gap-24 justify-center items-stretch ${
+            flipped ? 'md:flex-row-reverse' : 'md:flex-row'
+          }`}
+        >
+          <div
+            className={`flex flex-col gap-4 w-full md:w-2/5 lg:w-1/3 flex-shrink-0 ${
+              image ? '' : 'items-center'
+            }`}
+          >
+            <h2
+              className={`font-semibold text-5xl bg-clip-text text-transparent dark:text-transparent leading-normal ${
+                image ? '' : 'text-center'
+              }`}
+              style={{
+                backgroundImage: `linear-gradient(-70deg, ${end}, ${start})`,
+              }}
+            >
+              {title}
+            </h2>
+            <div className="text-lg text-gray-600 dark:text-gray-400 leading-7">
+              {description}
+            </div>
+          </div>
+          {image && (
+            <div
+              className="rounded-3xl overflow-hidden p-8 flex-grow flex flex-col justify-center items-stretch relative"
+              style={{
+                backgroundImage: `linear-gradient(70deg, ${start}, ${end})`,
+              }}
+            >
+              <Image
+                src={image}
+                className="rounded-xl"
+                layout="responsive"
+                width="100%"
+                height="100%"
+                alt={title}
+              />
+            </div>
+          )}
+        </div>
+        {children}
+      </div>
+    </FeatureWrapper>
+  )
+}
+
+function FeatureHighlights(props: {
+  textColor?: string
+  highlights?: Array<{
+    title: string
+    description: React.ReactNode
+    icon?: React.ReactNode
+  }>
+}) {
+  const { highlights, textColor } = props
+
+  const Highlight = {
```

Comment:
Renames to, no need to make it capitalized it not a class or react component
```suggestion
  const classes = {
```